releases/release-1.13/README.md

```diff
@@ -0,0 +1,139 @@
+
+## Notes About this Release
+
+* The Release Team will continue with the shortened Code Freeze, [see below](#conditionally-shortened-code-freeze).
+* The feature process is remaining as it has in prior releases, except the it is being renamed from features to Enhancements.
+* Enhancements that don't have complete code and tests by [Code Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.13/release-1.13.md#code-freeze) may be disabled by the release team before cutting the first beta.
+* The release team will escalate [release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) failures to SIGs throughout the cycle, not just near release cuts.
+* Unlike the previous cycles key deliverables, including the final release, do **not** fall on Tuesdays due to shortened release cycle.
+* The release cycle length is approximately only **10 weeks** long.
+
+## Timeline
+
+
+| **What** | **Who** | **Oct** | **Nov** | **Dec** | **Jan** | **DEV WEEK** | **TEST GATES** |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Start of Release Cycle | Lead | 1| | | | week 1 | |
+| Finalize Schedule | Lead | 5 | | | | | |
+| Begin collecting planned work from SIGs | Lead, Enhancements Lead | 8| | | | week 2 | |
+| Begin weekly release team meetings | Lead | 8 | | | | | |
+| Begin weekly status reports at Community | Lead, Shadow | 11 | | | | | |
+| Finalize Release Team | Lead | 12 | | | | | |
+| Start Release Notes Draft | Release Notes Lead | 16 | || | week 3 | |
+| Clean up enhancements repo | Enhancements Lead | 16 | | | | | |
+| "Enhancements Freeze" begins (EOD PST) | Enhancements Lead | 23 | || | week 4 | |
+| 1.13.0-alpha.1 release | Branch Manager | 23 | | | | | [master-blocking], [master-upgrade] |
+| Blog post: what we're working on for 1.13 | Communications | | 2 | | | week 5 | |
+| All release-1.9 CI jobs/testgrid removed | Test Infra Lead | | 6 | | | week 6 | |
+| Create 'release-1.13' branch and begin daily branch | Branch Manager | | 6 | | | | |
+| v1.13.0-alpha.0 | Branch Manager | | 6 | | | | |
+| 1.13.0-beta.0 released from branch | Branch Manager | | 6 | | | | [master-blocking], [master-upgrade] |
+| Docs deadline - Open placeholder PRs | Docs Lead | | 8 | | | | |
+| All release-1.13 CI jobs/testgrid created | Test Infra Lead | | 9 | | | | |
+| Begin Code Slush | Bot, Lead | | 9 | | | week 6 | |
+| Begin MWF Burndown meetings | Lead | | 12 | | | week 7 | |
```

Comment:
Could be sparsely attended?  I suggest you query ASAP and note privately the release team's travel plans for the two KubeCons (and anything else) so you know ahead of time who's going to be out of hand when.

Reply:
Thanks for the suggestion. Coming to think if it, igf I start weekly meetings from 10/1, I will move to MWF burndown a week earlier i.e., from Nov 5th.

Reply:
I'd reckon you're probably ok starting in week 7, if the milestone's work list is kept shorter and less ambitious in the feature definition phase.  And might not have too many people at KubeCon China...check and see.  Might be a non-issue.